variables.tf

```diff
@@ -56,20 +56,20 @@ variable "worker_pools" {
   }))
   default = [
     {
-      subnet_prefix    = "zone-1"
+      subnet_prefix    = "default"
```

Comment:
I would remove the defaults here. They do not really make sense for 2 reasons:
1. The subnet_prefix is making assuming on what is going to be passed in the vpc_subnets variable. That variable does not have default
2. The default subnet_prefix "subnet-X" that is used is still relatively confusing. It could be interpreted as creating 3 worker pools in 3 subnets, when this is actually not the case.

Reply:
Removed the default worker pool variable in the root module.